Ferret/CosmicExploration/WKSMissionInfomation.lua

```diff
@@ -20,8 +20,15 @@ function WKSMissionInfomation:wait_until_ready()
     self.ferret:wait_for_ready_addon("WKSMissionInfomation")
 end
 
+function WKSMissionInfomation:wait_until_done_crafting()
+    repeat
+        self.ferret:wait(0.1)
+    until not GetCharacterCondition(Conditions.Crafting40) and not GetCharacterCondition(Conditions.PreparingToCraft)
+end
+
```

Comment:
Could you move this to Character.lua and call it with `self.ferret.character:wait_until_done_crafting()`? Also could you bump the Ferret version by 1 patch?